### PR TITLE
Fixed liquibase issue in kork-sql-test

### PR DESCRIPTION
**Pull request by aman-agrawal**
## Summary
Project Jira : https://devopsmx.atlassian.net/browse/OP-20333
Project Doc : NA

Issue : SqlTestUtilTest TCs are failing bcoz there is issue in setting physicalFilePath
Solution : Added changelog-master.yml file and passed physicalFilePath in DatabaseChangeLog constructor

### How changes are verified
Liquibase testcase passing in intellij

## Documentation Updates
Do we need to update dashboards? No
Do we need to update SOP, new hire wiki or other documents? No

### Rollback, Deployment Details
Can this change be rolled back automatically without any issue?  Yes
Is this a backwards-compatible change in your opinion ?  Yes
Pre deployment steps : NA
Post deployment steps : NA

### CODE REVIEW

**Review by kirangodishala — COMMENTED** [review]
@aman-agrawal - a comment would help to understand why this change was required. Other than that it lgtm.

**Comment by aman-agrawal** [comment]
> @aman-agrawal - a comment would help to understand why this change was required. Other than that it lgtm.

Initially we are getting changeLog as Null in initDatabase(String jdbcUrl, SQLDialect dialect, String dbName) method and it is causing issue for Liquibase.update(dbName). See update() method in Liquibase.java.
With this change, it is set to "db/changelog-master.yml" and Liquibase.update(dbName) is successful.

File: kork-sql-test/src/main/java/com/netflix/spinnaker/kork/sql/test/SqlTestUtil.java [hunk 1]
```diff
@@ -222,7 +222,7 @@ public static TestDatabase initDatabase(String jdbcUrl, SQLDialect dialect, Stri
 
     Liquibase migrate;
     try {
-      DatabaseChangeLog changeLog = new DatabaseChangeLog();
+      DatabaseChangeLog changeLog = new DatabaseChangeLog("db/changelog-master.yml");
```
Comment (kirangodishala) [inline kork-sql-test/src/main/java/com/netflix/spinnaker/kork/sql/test/SqlTestUtil.java:225]: @aman-agrawal - can you add comment above this line describing why this change in constructor is needed now (like the new liquibase version does null checks etc).

Comment (aman-agrawal) [reply kork-sql-test/src/main/java/com/netflix/spinnaker/kork/sql/test/SqlTestUtil.java:227]: Done

**Review by kirangodishala — APPROVED** [review]

**Review by sanopsmx — APPROVED** [review]
